packages/openapi-generator/src/options/options-per-service.ts

```diff
@@ -1,11 +1,20 @@
 import { existsSync, promises } from 'fs';
+import { parse } from 'path';
+import { UniqueNameGenerator } from '@sap-cloud-sdk/util';
+import { AbsoluteAndRelativePath } from '../generator';
+import { ParsedGeneratorOptions } from './generator-options';
 
 const { readFile } = promises;
 
 /**
  * Represents the service options for all services, mapped from the input file path to the service configuration.
  */
-export type OptionsPerService = Record<string, Partial<ServiceOptions>>;
+export type OptionsPerService = Record<string, ServiceOptions>;
```

Comment:
I understand that this is the actual return type of `getOptionsPerService`, but is this ever relevant? After it was created we never really use it and it might be confusing to have two types here.

Reply:
I removed the export for the partial config since it is only used internally.

Reply:
That doesn't really address my concern, but it is minor anyway. Let's leave it at that 😊